pkg/pod/entrypoint_lookup_impl_test.go

```diff
@@ -0,0 +1,77 @@
+package pod
```

Comment:
We'll need a copyright header on this new file.

Reply:
Thank you for reminding! 👍